### ISSUE TITLE
[bug] Unintentional wrap of button label for "SaveChanges" (multiple languages)

### ISSUE THREAD

**Issue by bluemoehre** [jellyfin/jellyfin-web#6211]
**Describe The Bug**

The label of the save button breaks into multiple lines. This unnecessarily affects the layout and wastes space on mobile screens. 
It happens with several languages if the label has multiple words and exceeds ~16 chars.

[All related project files (59)](https://github.com/search?q=repo%3Ajellyfin%2Fjellyfin-web+SaveChanges&type=code)

(This is an UI/UX issue only - no functional problems detected yet)

**Steps To Reproduce**

1. Change UI language to eg. `DE` |  `FR` | `HU` | `RO` …
2. Go to Overview
3. Open any item's menu
4. Select Edit Metadata

**Expected Behavior**

Button label _should not break_ into multiple lines. This can be achieved by using shorter text, enlarging the buttons or allow them to grow.

At least for `DE`, `EN`, `ES`, `FR` I can say that **a single word** like "Save" or "Apply" **is absolutely common**.
I am sure it's the same with more languages, so **I would recommend updating the translations**.

**EDIT:** I found there is already a short version `Save` in most languages. I am not sure why this is mixed.

**Screenshots**

![Screenshot](https://github.com/user-attachments/assets/b83613ab-79b9-41d1-8fd7-c78967a525c2)

**System (please complete the following information):**
 - Platform: all
 - Browser: all
 - Jellyfin Version: 10.9.11

**Additional Context**

I've already checked the CSS, and if we prefer changes here, it's also possible. Just a slightly higher risk of breaking something elsewhere.

**Comment by tylers-username** [comment]
MR: https://github.com/jellyfin/jellyfin-web/pull/6250

**Comment by bluemoehre** [comment]
Bump

**Comment by thornbill** [comment]
Is this still an issue since your PR #6212 changed the text to use the shorter "Save" text?

**Comment by bluemoehre** [comment]
Yes it's fixed for DE. I did not check the other languages yet and did not want to rely on a guess.

**Comment by bluemoehre** [comment]
Looks good to me. Thanks everyone.
